src/hipscat/pixel_math/filter.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
```

Comment:
=D

Thank you for extracting out the common logic! This is a nice cleanup!

Reply:
Yeah, I really like this :)